src/test/java/org/folio/service/processing/kafka/WriteStreamWrapperTest.java

```diff
@@ -0,0 +1,131 @@
+package org.folio.service.processing.kafka;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.kafka.client.producer.KafkaProducer;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+//import org.junit.jupiter.api.Assertions;
```

Comment:
You should not include commented code in PR

Reply:
It's about the whole class

Reply:
@viktor-draban Thx! But we'll remove this class at all

Reply:
Thank you, it is fixed